gsoc-2024-proxy/src/main/java/proxy/ProxyVerticle.java

```diff
@@ -15,6 +22,24 @@ public static void main(String[] args) {
 
   @Override
   public void start(Promise<Void> startPromise) {
+    HttpClient proxyClient = vertx.createHttpClient();
+
+    HttpProxy httpProxy1 = HttpProxy.reverseProxy(proxyClient);
+    httpProxy1.origin(8080, "localhost");
+
+    HttpServer proxyServer = vertx.createHttpServer();
+    Router proxyRouter = Router.router(vertx);
+    proxyRouter.route()
+        .handler(
+          CorsHandler.create()
+            .addRelativeOrigin("http://localhost:3000")
+            .allowedMethod(HttpMethod.GET)
+        );
```

Comment:
Why is Cors needed? Your Proxy should send the requests for `/joke` to the backend service, and the others to the website service.

Reply:
sorry for that. i misunderstand that proxy is for only backed server. so cors needed for fronted. now i will modify for frondend proxy.